dibi/libs/DibiFluent.php

```diff
@@ -315,7 +315,7 @@ public function execute($return = NULL)
 	 */
 	public function fetch()
 	{
-		if ($this->command === 'SELECT') {
+		if ($this->command === 'SELECT' && $this->_export('limit') === NULL) {
```

Comment:
Why is your test different than this https://github.com/dg/dibi/commit/ba4cbd9df3724a8c91b3fe7c129e2f09cc855372?